app/views/course/assessment/assessments/_assessment.html.slim

```diff
@@ -33,7 +33,11 @@
         = link_to course_achievement_path(current_course, achievement) do
           = display_achievement_badge(achievement)
   td.table-start-at
-    - if condition_not_satisfied(can_attempt_assessment, assessment_with_loaded_timeline, assessment_time)
+    - if assessment_not_unlocked(assessment_time)
+      div.condition-not-satisfied data-toggle='tooltip' title="#{t('.assessment_not_unlocked')}"
+        = render partial: 'course/lesson_plan/items/personal_or_ref_time',
+                 locals: { item: @items_hash[assessment.id], course_user: current_course_user, attribute: :start_at, datetime_format: :short }
+    - elsif condition_not_satisfied(can_attempt_assessment, assessment_with_loaded_timeline, assessment_time)
       div.condition-not-satisfied data-toggle='tooltip' title="#{t('.condition_not_satisfied')}"
         = render partial: 'course/lesson_plan/items/personal_or_ref_time',
                  locals: { item: @items_hash[assessment.id], course_user: current_course_user, attribute: :start_at, datetime_format: :short }
```

Comment:
There's a bit of duplicated code here when using the if/else statements to conditionally render the correct tooltip message, and I'm wondering if there's a cleaner way to approach this?